app/libs/globals.php

```diff
@@ -22,20 +22,19 @@
 
 preg_match('(\/.+\/)', $_SERVER['SCRIPT_FILENAME'], $match);
 
-$path = str_replace('/var/www/html', '', $match[0]);
-$path = str_replace('/admin/', '/', $path);
-$path = str_replace('/api/', '/', $path);
-$path = str_replace('/auth_cookie/', '/', $path);
-$path = str_replace('/auth_domain/', '/', $path);
-$path = str_replace('/auth_token/', '/', $path);
-$path = str_replace('/dynicons/', '/', $path);
-$path = str_replace('/login/', '/', $path);
-$path = str_replace('/qrcode/', '/', $path);
-$path = str_replace('/scripts/', '/', $path);
-$path = str_replace('sources/../mailer', '', $path);
-$path = str_replace('/utils/', '/', $path);
+$folders = array('admin', 'api', 'auth.cookie', 'auth_domain', 'auth_token', 'dynicons', 'login', 'qrcode', 'scripts', 'utils', 'sources', '..', 'mailer', 'var', 'www', 'html', '');

```

Comment:
Please add some comments here to explain to future devs what exactly this array is for.  And I know it's more for show, but put each on a separate line.  Others may want to add comments per path.  Let this be one of the new methods we can point to as "this is how to do it."

To include a "why" we're doing it this way.